tests/stencils/test_fields.py

```diff
@@ -81,3 +82,23 @@ def hv_offsets(
             a = b[Edge, k] + c
             a = b[Edge > Cell > Edge, k + 1] + c[Edge > Cell > Edge, k]
             a = b[Edge, k] + b[Edge, k - 1] + c[Edge > Cell > Edge]
+
+
+@stencil
+def non_sensical_vertical_index_in_2d_field(
```

Comment:
again, this test will fail soon, c.f. [PR](https://github.com/MeteoSwiss-APN/dawn/pull/1034)